docs/content/development/extensions-contrib/orc.md

```diff
@@ -1,113 +0,0 @@
----
-layout: doc_page
-title: "ORC"
----
-
-<!--
-  ~ Licensed to the Apache Software Foundation (ASF) under one
-  ~ or more contributor license agreements.  See the NOTICE file
-  ~ distributed with this work for additional information
-  ~ regarding copyright ownership.  The ASF licenses this file
-  ~ to you under the Apache License, Version 2.0 (the
-  ~ "License"); you may not use this file except in compliance
-  ~ with the License.  You may obtain a copy of the License at
-  ~
-  ~   http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing,
-  ~ software distributed under the License is distributed on an
-  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  ~ KIND, either express or implied.  See the License for the
-  ~ specific language governing permissions and limitations
-  ~ under the License.
-  -->
-
-# ORC
-
-To use this extension, make sure to [include](../../operations/including-extensions.html) `druid-orc-extensions`.
-
-This extension enables Druid to ingest and understand the Apache ORC data format offline.
-
-## ORC Hadoop Parser
-
-This is for batch ingestion using the HadoopDruidIndexer. The inputFormat of inputSpec in ioConfig must be set to `"org.apache.hadoop.hive.ql.io.orc.OrcNewInputFormat"`.
-
-|Field     | Type        | Description                                                                            | Required|
-|----------|-------------|----------------------------------------------------------------------------------------|---------|
-|type      | String      | This should say `orc`                                                                  | yes|
-|parseSpec | JSON Object | Specifies the timestamp and dimensions of the data. Any parse spec that extends ParseSpec is possible but only their TimestampSpec and DimensionsSpec are used. | yes|
-|typeString| String      | String representation of ORC struct type info. If not specified, auto constructed from parseSpec but all metric columns are dropped | no|
```

Comment:
Looks like `typeString` is no longer a thing, because things are being detected better, and automatically now. But could this cause anybody to be sad, that may have been manually specifying an override typeString for some purpose?

The removal of typeString should be called out in the docs & release notes, or perhaps readded, if feasible and useful. (I am not sure about either of those two!)

Reply:
Added a section to the extension docs about "migrating from the 'contrib' extension", with what needs changed and how to change it to maintain ingested schema, along with some better examples and improved explanation of stuff.